seq2science/rules/gene_counts.smk

```diff
@@ -153,6 +157,21 @@ else:
                 counts.index.name = "gene"
                 counts.to_csv(output[0], sep="\t")
 
+    rule tpm_matrix:
+        """
+        Create a TPM table from a counts table.
+        """
+        input:
+            cts=rules.count_matrix.output,
+            gtf=expand("{genome_dir}/{{assembly}}/{{assembly}}.annotation.gtf",**config),
+        output:
+            tpms = expand("{counts_dir}/{{assembly}}-TPM.tsv",**config),
+            lengths = expand("{counts_dir}/{{assembly}}-gene_lengths.tsv",**config),
+        log:
```

Comment:
maybe add a explain message here? It's perhaps non-trivial by what/how the tpm was calculated